backend/src/Entity/Course.php

```diff
@@ -60,12 +60,20 @@ class Course
     #[ORM\OneToMany(mappedBy: 'course', targetEntity: CourseComment::class, orphanRemoval: true)]
     private Collection $courseComments;
 
+    // This is the self-referential many-to-many relationship
+    #[ORM\ManyToMany(targetEntity: self::class, inversedBy: "IdenticalCourses")]
+    #[ORM\JoinTable(name: "identical_courses")]
+    #[ORM\JoinColumn(name: "course_id", referencedColumnName: "id")]
+    #[ORM\InverseJoinColumn(name: "identical_course_id", referencedColumnName: "id")]
```

Comment:
Why these last 3 lines?
You could do the same thing as with oldcoruses or newCoruses

Reply:
I pushed the wrong part that was a copy paste of something I found but later changed it. I'll repush it and change its location to admin.